### PR TITLE
Please review contact form remote pair programming

### PR DESCRIPTION
**Pull request by flov3rh**
-Add contact form with phone version and min desktop version
-Implemented feature with collaborative remote pair programming

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,5 +176,33 @@ <h3>Skills</h3>
...
+    <footer>
+      <a href="/" class="footer-mail">floperh@gmail.com</a>
+      <ul class="list-format font-style footer-icon">
+        <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+        <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+        <li><a href="#" class="fa-brands fa-angellist"></a></li>
+        <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+        <li><a href="mailto: floperh@gmail.com" class="fa-solid fa-envelope"></a></li>
+      </ul>
+    </footer>
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please stick to the design as much as possible like font, colours, images, and spacing (margin/padding). As I notice that you design is missing the small background image please implement it and also I would suggest you please download the social media icons from Figma design and use it.
- [x] Also make sure that your font matches the design as it has more font weight for your email.
### Your design
his

### Figma design
fimga

File: index.html [hunk 2]
```diff
@@ -176,5 +176,33 @@ <h3>Skills</h3>
...
     </section>
+
+    <section id="form-section">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+
+      <form class="" action="index.html" method="post">
+        <div class="fullname">
```
Comment (roshan-bajgain) [inline index.html]: - [ ] According to the project requirement you should use the [formspree service](https://formspree.io/html).[formspree service](https://formspree.io/html) that uses the `action` and `method` attributes. 
`* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`
Make sure the form is submitted successfully.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @flov3rh and @abdullah97-pro, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [x] [Optional] Please make sure that you commit message should be `capatilized` first letter of the setence should be `capatilized`. Keep in mind for next project.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @flov3rh,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

- [x] Kindly consider working on optional suggestion

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**